### PR TITLE
Mike/melted dump to pooh/at 387

### PR DESCRIPTION
**Pull request by mikehow1984**
#### What's this PR do?

- moves the `meltedDataDumpToCSV.js` job from `jobs/training` to `tasks` to dump CSV to Pooh
- pulls data from `autoForceMelt.js` into a json file that is converted into a CSV to dump into Pooh

#### How should this be manually tested?
comment out everything in `autoForceMelt.js` that actually force melts users. It should produce `poohMeltedData.json`. Then run `forceMeltDataToPooh.js`, which should produce a CSV from that data.

For the `meltedDataDumpToCSV.js`, run it and it should produce a CSV from the data.

### CODE REVIEW

File: jobs/training/autoForceMelt.js [hunk 1]
```diff
@@ -82,9 +86,10 @@ function compileCourseGradeData(courseId) {
 function compileSubmissionEvents({ quizEvents, assignmentEvents, fullRoleplayId, riskAssessmentId, courseId }) {
   const submissionLogs = [fullRoleplayId, riskAssessmentId]
   .map(assignmentId => canvas.getAllSubmissions(courseId, assignmentId));
+  
   return Promise.all(submissionLogs).then(submissionEvents => (
-    { quizEvents, assignmentEvents, fullRoleplayId, riskAssessmentId, submissionEvents }
-  ));
+    { quizEvents, assignmentEvents, fullRoleplayId, riskAssessmentId, courseInfo, submissionEvents }
```
Comment (apackin) [inline jobs/training/autoForceMelt.js]: What is courseInfo here?

File: jobs/training/autoForceMelt.js [hunk 2]
```diff
@@ -221,6 +253,12 @@ function createTemplateName(messageData) {
       templateName += '-melt';
       messageData.cc1 = ctlOnlineRoles[CONFIG.CTLOnline.roles['Crisis Counselor I']];
       checkEnrollmentAndMeltUser(messageData);
+      
+      const poohData = messageData.meltData;
+      poohData.email = traineeEmail;
+      poohData.melted_timestamp = new Date(Date.now()).toISOString();
+      poohData.deferred = poohData.deferred ? true : false;
```
Comment (apackin) [inline jobs/training/autoForceMelt.js]: The ternary here based on poohData.deferred should be messageData.deferred?

Comment (mikehow1984) [inline jobs/training/autoForceMelt.js]: Correct. I originally made poohData = messageData and didn't also change this.

File: jobs/training/autoForceMelt.js (hunk 1 above)
Comment (mikehow1984) [inline jobs/training/autoForceMelt.js]: Probably something I missed when I resolved conflicts. Deleting this.

**Pull request merged**